src/Features/Core/Portable/Diagnostics/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -95,6 +95,8 @@ internal static class IDEDiagnosticIds
         public const string UseIndexOperatorDiagnosticId = "IDE0056";
         public const string UseRangeOperatorDiagnosticId = "IDE0057";
 
+        public const string MakeStructFieldsWritable = "IDE0058";
```

Comment:
>IDE0058 [](start = 56, length = 7)

This has merge conflict now: [http://source.roslyn.io/#Microsoft.CodeAnalysis.Features/Diagnostics/Analyzers/IDEDiagnosticIds.cs,98](http://source.roslyn.io/#Microsoft.CodeAnalysis.Features/Diagnostics/Analyzers/IDEDiagnosticIds.cs,98)

Reply:
@mavasani 
Hi, I resolved conflicts and chose `IDE0062` as id for analyzer